proposed/clock-meta.md

```diff
@@ -83,16 +65,16 @@ final class SystemClock implements \Psr\Clock\ClockInterface
 
 final class UTCClock implements \Psr\Clock\ClockInterface
 {
-    private TimeZoneAwareClock $inner;
+    private \DateTimeZone $utcTimeZone;
```

Comment:
Is there a particular reason to not use a proper `use` statement on the top of the file?

Reply:
Its not really a file, just some example implementations, this makes they very copy+paste-able.

Reply:
Ok fair enough.